run

```diff
@@ -333,6 +333,12 @@ setup() {
     apt-get install --no-install-recommends -y libv4l-dev
   fi
 
+  # git
+  git_version=$(dpkg --status git | grep -Po '^Version: \K[^-]*')
+  if [[ $git_version == "" ]]; then
```

Comment:
Could we simplify to `which git`? Seems OK if git is already available in `PATH` and not necessarily installed via dpkg.